app/routes/app/brainstorm/$marketplace.topic.new.tsx

```diff
@@ -0,0 +1,45 @@
+export default function CreateTopic() {
+  return (
+    <div className="flex">
+      <div className="flex flex-col basis-2/3 mx-auto">

```

Comment:
You can use container so we have consistent margins across our pages
https://tailwindcss.com/docs/container

Also no need to use a flex here